### DISCUSSION TITLE
DAI 2023-2024 - Check out my GitHub profile! - @KevinFarine

### DISCUSSION THREAD

**Discussion by KevinFarine** [heig-vd-dai-course/heig-vd-dai-course#15] [Show and tell]
https://github.com/KevinFarine

**Comment by ludelafo** [comment]
Great README! What is LFS? The only LFS I know is Linux From Scratch (<https://www.linuxfromscratch.org/lfs/>).

**Reply by KevinFarine** [reply]
LSF, in my case, means "Langue des Signes Français"
I'll check your link for sure.

**Reply by ludelafo** [reply]
Oh cool! I would love to learn sign language myself, one day maybe!
